core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -402,6 +402,6 @@ object TopicCommand extends Logging {
       s"- value set here: $maxMessageBytes\n" +
       s"- Default Broker replica.fetch.max.bytes: ${kafka.server.Defaults.ReplicaFetchMaxBytes}\n" +
       s"- Default Broker max.message.bytes: ${kafka.server.Defaults.MessageMaxBytes}\n" +
-      s"- Default Consumer fetch.message.max.bytes: ${ConsumerConfig.FetchSize}\n\n"
+      s"- Default Consumer max.partition.fetch.bytes: ${ConsumerConfig.FetchSize}\n\n"
```

Comment:
`ConsumerConfig.MAX_PARTITION_FETCH_BYTES_CONFIG`'s default should be used from ConsumerConfig, right?